elevate-project/constants/routes.js

```diff
@@ -1260,6 +1260,16 @@ module.exports = {
 				type: 'GET'
 			},
 		},
+		{
```

Comment:
@borkarsaish65 in the service  you added post method please make  it consistent. if we are not expecting any req body for api call . make it GET in service